pkgs/tools/misc/findutils/default.nix

```diff
@@ -24,7 +24,12 @@ stdenv.mkDerivation rec {
     substituteInPlace xargs/xargs.c --replace 'char default_cmd[] = "echo";' 'char default_cmd[] = "${coreutils}/bin/echo";'
   '';
 
-  patches = [ ./no-install-statedir.patch ];
+  patches = [
+    # Skip trying to create install directories:
+    # 1. We already have them
+    # 2. This installer is trying to create /var/cache
+    ./no-install-statedir.patch
```

Comment:
How about removing the patch altogether and using "substituteInPlace" to do the replacement. The comments can be kept, maybe adjusted a bit.

The reasoning is that the patch does only one change and that can be easily done with "substituteInPlace".